### DISCUSSION TITLE
Question about game not rendering at inactive tab

### DISCUSSION THREAD

**Discussion by erhanbaris** [pixijs/pixijs#8369] [Q&A]
Hello
Thank you effort make that kind of wonderful open source project.
I could not find any good answer about keep rendering at inactive tab. There are couple of guy shared github links but those links not working. Also some of the answer are very high level. Any one know how to do that?

### Environment

- **`pixi.js` version**:6.3.2

**Comment by domis86** [comment]
Hi,
Please check this issue: https://github.com/pixijs/pixijs/issues/7497

**Reply by erhanbaris** [reply]
Thanks for your answer but I could not understand how to solve the issue.

**Reply by domis86** [reply]
As you see in above linked issue is not a problem with Pixi.js but with browser mechanics in general.  You should try finding solution related to that.

Please also see:
* https://www.html5gamedevs.com/topic/21557-keep-requestanimationframe-running-in-the-background/
* https://stackoverflow.com/questions/5927284/how-can-i-make-setinterval-also-work-when-a-tab-is-inactive-in-chrome
